src/entries/popup/pages/home/Activity/ActivitiesList.tsx

```diff
@@ -44,6 +44,22 @@ import { ActivityTypeIcon } from './ActivityTypeIcon';
 import { ActivityValue } from './ActivityValue';
 import { NoActivity } from './NoActivity';
 
+function getNftChangesTag(transaction: RainbowTransaction): string | undefined {
+  const { asset, changes } = transaction;
+  const nftChangesAmount =
+    changes?.reduce((count, c) => {
+      if (c && asset?.address === c.asset.address && c.asset.type === 'nft') {
+        const numericValue = Number.isInteger(Number(c.value))
+          ? Number(c.value)
+          : 1;
```

Comment:
Curious if this will break when backend fixes this bad data
What do we want to request instead?